src/Admin/AbstractAdmin.php

```diff
@@ -588,6 +588,12 @@ public function initialize()
      */
     public function configure()
     {
+        foreach ($this->getExtensions() as $extension) {
```

Comment:
I would move this code inside the `initialize` method instead.

Reply:
Sure, after `$this->configure()` ?

Reply:
yes